js_modules/dagit/packages/core/src/app/AppError.tsx

```diff
@@ -12,34 +12,61 @@ interface DagsterGraphQLError extends GraphQLError {
 
 const ErrorToaster = Toaster.create({position: 'top-right'});
 
-const showGraphQLError = (error: DagsterGraphQLError, operationName?: string) => {
+const showGraphQLError = (
+  error: DagsterGraphQLError,
+  operationName?: string,
+  networkPathPrefix?: string,
+) => {
   const message = (
     <div>
       Unexpected GraphQL error
-      <AppStackTraceLink error={error} operationName={operationName} />
+      <AppStackTraceLink
+        error={error}
+        operationName={operationName}
+        networkPathPrefix={networkPathPrefix}
+      />
     </div>
   );
   ErrorToaster.show({message, intent: 'danger'});
   console.error('[GraphQL error]', error);
 };
 
-export const errorLink = onError((response: ErrorResponse) => {
-  if (response.graphQLErrors) {
-    const {graphQLErrors, operation} = response;
-    const {operationName} = operation;
-    graphQLErrors.forEach((error) => showGraphQLError(error as DagsterGraphQLError, operationName));
-  }
-  if (response.networkError) {
-    console.error('[Network error]', response.networkError);
-  }
-});
+export const errorLink = (pathPrefix: string) =>
+  onError((response: ErrorResponse) => {
+    if (response.graphQLErrors) {
+      const {graphQLErrors, operation} = response;
+      const {operationName} = operation;
+      graphQLErrors.forEach((error) =>
+        showGraphQLError(error as DagsterGraphQLError, operationName, pathPrefix),
+      );
+    }
+    if (response.networkError) {
+      console.error('[Network error]', response.networkError);
+    }
+  });
 
 interface AppStackTraceLinkProps {
   error: DagsterGraphQLError;
   operationName?: string;
+  networkPathPrefix?: string;
 }
 
-const AppStackTraceLink = ({error, operationName}: AppStackTraceLinkProps) => {
+const AppStackTraceLink = ({error, operationName, networkPathPrefix}: AppStackTraceLinkProps) => {
+  const [versions, setVersions] = React.useState<
+    {dagit_version: string; dagster_graphql_version: string; dagster_version: string} | undefined
+  >();
+
+  React.useEffect(() => {
+    const loadVersions = async () => {
+      const versionsResponse = await fetch(`${networkPathPrefix || ''}/dagit_info`, {
+        method: 'GET',
+        credentials: 'include',
+      });
+      setVersions(await versionsResponse.json());
+    };
+    loadVersions();
```

Comment:
Could we wrap this call in a try/catch that just logs the error and does nothing? I'm worried about potentially breaking the display of errors in some strange scenario we haven't thought of (if the network is down or timing out, etc. and this fetch fails)